### ISSUE TITLE
Unable install or update package

### ISSUE THREAD

**Issue by justutiz** [vyuldashev/nova-permission#165]
![image](https://user-images.githubusercontent.com/6322926/123286051-3e878e00-d516-11eb-9618-70942dd26414.png)

```
Using GitHub token authentication
Downloading https://api.github.com/repos/rvxlab/nova-permission
[404] https://api.github.com/repos/rvxlab/nova-permission
Executing command (CWD): git config github.accesstoken
Downloading https://api.github.com/repos/rvxlab/nova-permission
[404] https://api.github.com/repos/rvxlab/nova-permission


  [Composer\Downloader\TransportException]
  The "https://api.github.com/repos/rvxlab/nova-permission" file could not be downloaded (HTTP/2 404 )

```

**Comment by RVxLab** [comment]
@justutiz You probably already fixed this but with the tagging of 3.0.1 I deleted that repo.

**Comment by justutiz** [comment]
> @justutiz You probably already fixed this but with the tagging of 3.0.1 I deleted that repo.

yeah, already fixed :) thanks
